util/testutil/tar.go

```diff
@@ -122,8 +125,58 @@ func BuildTarGz(ents []TarEntry, compressionLevel int, opts ...BuildTarOption) i
 	return pr
 }
 
+// WriteTarGz writes the contents of a tar (gz) to a specified path(targz file)
+func WriteTarGz(targzName string, r io.Reader) (*string, error) {
```

Comment:
The func is not limited to `TarGz` right? (e.g., both return of `BuildTar` and `BuildTarGz` can be passed to this func which writes to a file). If so suggest to change name to `WriteTarToTempFile`)

Reply:
also can we change the return type to `string` instead of `*string`?

Reply:
Sure